docs/helm.md

```diff
@@ -0,0 +1,37 @@
+# osc-cloud-controller-manager
+
+![Version: 0.1.1](https://img.shields.io/badge/Version-0.1.1-informational?style=flat-square) ![AppVersion: 0.1.1](https://img.shields.io/badge/AppVersion-0.1.1-informational?style=flat-square)
+
+A Helm chart for OSC CCM cloud provider
+
+**Homepage:** <https://github.com/outscale-dev/cloud-provider-osc/>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| 3DS Outscale | <support@outscale.com> |  |
+
+## Source Code
+
+* <https://github.com/outscale-dev/cloud-provider-osc/>
+
+## Requirements
+
+Kubernetes: `>=1.14.0-0`
```

Comment:
Is it still true ?

Reply:
Based on kubernetes api deprecation:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/

Reply:
At least 1.22.0 based on what i found here:
https://github.com/outscale-dev/cloud-provider-osc/blob/OSC-MIGRATION/deploy/k8s-osc-ccm/templates/osc-ccm.yaml
(The rbac.authorization.k8s.io/v1beta1 API version of ClusterRole, ClusterRoleBinding, Role, and RoleBinding is no longer served as of v1.22.)

Reply:
You can also change it on csi ?

Reply:
Good point, I'll make another PR for that

Reply:
Ok